examples/main.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
```

Comment:
a notebook is probably better since it is rendered on GitHub one does not need to clone and execute the script to see the outcome

Reply:
Notebook will follow in another PR.